### PR TITLE
Several changes on Kontlin.

### PR DESCRIPTION
**Pull request by weakish**
- anonymous function -> lambda expression: they are different in Kotlin.
- change coding style according to Kotlin Reference.
- Remove remove unnecessary semicolons.
- Replace code block language `java` with `kotlin`.

---
kotlin.html.markdown | 37 +++++++++++++++++++------------------
1 file changed, 19 insertions(+), 18 deletions(-)

### CODE REVIEW

File: kotlin.html.markdown [hunk 1]
```diff
@@ -2,14 +2,15 @@
 language: kotlin
 contributors:
     - ["S Webber", "https://github.com/s-webber"]
+    - ["Jakukyo Friel", "https://weakish.github.io"]
```
Comment (geoffliu) [inline kotlin.html.markdown]: While we do appreciate your contribution, but please do not add yourself to contributors for only formatting changes.

File: kotlin.html.markdown [hunk 2]
```diff
@@ -119,16 +120,16 @@ fun helloWorld(val name : String) {
...
     }
     // Named functions can be specified as arguments using the :: operator.
     val notOdd = not(::odd)
     val notEven = not(::even)
-    // Anonymous functions can be specified as arguments.
+    // Lambda expressions can be specified as arguments.
     val notZero = not {n -> n == 0}
     /*
-    If an anonymous function has only one parameter
+    If a Lambda has only one parameter
```
Comment (geoffliu) [inline kotlin.html.markdown]: Lowercase `l`

File: kotlin.html.markdown [hunk 3]
```diff
@@ -169,7 +170,7 @@ fun helloWorld(val name : String) {
     println(fooData) // => DataClassExample(x=1, y=2, z=4)
 
     // Data classes have a "copy" function.
-    val fooCopy = fooData.copy(y = 100)
+    val fooCopy = fooData.copy(y=100)
```
Comment (geoffliu) [inline kotlin.html.markdown]: At least on the Kotlin website, they keep the spaces in function calls.

Comment (geoffliu) [inline kotlin.html.markdown]: https://kotlinlang.org/docs/reference/data-classes.html#copying

Comment (weakish) [inline kotlin.html.markdown]: Thanks a lot. Python uses `parameter=default_value` style. I did not noticed kotlinlang.org uses a different style.

**Comment by vendethiel** [comment]
@weakish Is that good to merge?

**Comment by weakish** [comment]
@vendethiel Yes.

**Comment by vendethiel** [comment]
Help

**Pull request merged**